README.rst

```diff
@@ -107,6 +107,37 @@ Installing this package will install ``nvidia-dali`` and its dependencies, if th
 
 ----
 
+Compiling DALI from source (Docker) - recommended
+-------------------------------------------------
+
+Prerequisites
+^^^^^^^^^^^^^
+
+.. |docker link| replace:: **Docker**
+.. _docker link: https://docs.docker.com/install/
+
+.. table::
+   :align: center
+
+   +----------------------------------------+---------------------------------------------------------------------------------------------+
+   | Linux x64                              |                                                                                             |
+   +----------------------------------------+---------------------------------------------------------------------------------------------+
+   | |docker link|_                         | Follow installation guide and manual at the link.                                           |
```

Comment:
Instead of this slightly meaningless sentence, we could specify, that docker version 17.05 is required (for multi-stage build).
I've put this in my previous PR for README, but since we're not going to add this in nearest future, it might be here.

Reply:
Done